components/desktop/mate/mate-applets/manifests/sample-manifest.p5m

```diff
@@ -23,29 +23,29 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 file path=usr/bin/mate-invest-chart
-file path=usr/lib/mate/accessx-status-applet
-file path=usr/lib/mate/command-applet
-file path=usr/lib/mate/invest-applet
-file path=usr/lib/mate/mate-charpick-applet
-file path=usr/lib/mate/mate-drivemount-applet
-file path=usr/lib/mate/mate-geyes-applet
-file path=usr/lib/mate/mate-multiload-applet
-file path=usr/lib/mate/mate-netspeed-applet
-file path=usr/lib/mate/mateweather-applet
-file path=usr/lib/mate/stickynotes-applet
-file path=usr/lib/mate/timer-applet
-file path=usr/lib/mate/trashapplet
-file path=usr/lib/python2.7/vendor-packages/mate_invest/__init__.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/about.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/applet.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/chart.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/currencies.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/defs.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/help.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/networkmanager.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/preferences.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/quotes.py
-file path=usr/lib/python2.7/vendor-packages/mate_invest/widgets.py
+file path=usr/lib/$(MACH64)/mate/accessx-status-applet
+file path=usr/lib/$(MACH64)/mate/command-applet
+file path=usr/lib/$(MACH64)/mate/invest-applet
+file path=usr/lib/$(MACH64)/mate/mate-charpick-applet
+file path=usr/lib/$(MACH64)/mate/mate-drivemount-applet
+file path=usr/lib/$(MACH64)/mate/mate-geyes-applet
+file path=usr/lib/$(MACH64)/mate/mate-multiload-applet
+file path=usr/lib/$(MACH64)/mate/mate-netspeed-applet
+file path=usr/lib/$(MACH64)/mate/mateweather-applet
+file path=usr/lib/$(MACH64)/mate/stickynotes-applet
+file path=usr/lib/$(MACH64)/mate/timer-applet
+file path=usr/lib/$(MACH64)/mate/trashapplet
+file path=usr/lib/python2.7/vendor-packages/64/mate_invest/__init__.py
```

Comment:
I suppose, this is wrong. Python scripts can go just to usr/lib/python2.7/vendor-packages/mate_invest/

Reply:
OK I will fix it.